builtin/providers/postgresql/config.go

```diff
@@ -70,7 +81,7 @@ func (c *Config) NewClient() (*Client, error) {
 func (c *Client) Connect() (*sql.DB, error) {
 	db, err := sql.Open("postgres", c.connStr)
 	if err != nil {
-		return nil, fmt.Errorf("Error connecting to PostgreSQL server: %s", err)
+		return nil, fmt.Errorf("Error connecting to PostgreSQL server: %v", err)
```

Comment:
We should switch all of these to use the `err wrap.Wrapf` pattern as we have been rolling through the rest of the code base.